### ISSUE TITLE
[BUG] Missing Sounds

### ISSUE THREAD

**Issue by FriskTheFallenHuman** [kolessios/insource-legacy#10]
i have mounted L4D2 and His DLC and hl2 but there's other missing sound for the weapon

- the pistol his firing and reloading sound is missing
- the ak47 just have the reload sound
- the shotgun just have the firing sound
- and for the Sniper and M16 they don't play they have the missing sounds

also for the impact sounds

**Comment by kolessios** [comment]
Yep, let's see what I can do about it, I have the sound configuration made a mess and some sounds I do not think I can make public.

**Comment by FriskTheFallenHuman** [comment]
I suggest you use for the sounds of impact, those of the HL2 and the weapons of the L4D2

**Comment by kolessios** [comment]
These are the sounds I have now, from L4D2, Counter Srike Global Offensive and other sounds under license.

https://drive.google.com/open?id=0B9QgMNC5et3nVWxNaGRSNEZ4RVE

**Note:** Each weapon has 2 sounds for its shot. One of them is normal and is used for the firing sound of the local player, the other uses the system within Source for [distant sounds](https://developer.valvesoftware.com/wiki/Soundscripts#Distance_variance_in_Source) and is used for the shots of other players.  (Left channel: Close sound, Right channel: Far sound)

**Comment by FriskTheFallenHuman** [comment]
Gogle Drive: Access Denied

**Comment by kolessios** [comment]
I'm sorry, they're already public.

**Comment by FriskTheFallenHuman** [comment]
also someone footsteps are missings

**Comment by FriskTheFallenHuman** [comment]
for the metal
